<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup_menu</class>
 <widget class="QDialog" name="signup_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbBoarderSignUp">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>741</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Rectangle 1 */


position: absolute;
width: 794px;
height: 491px;
left: 34px;
top: 112px;

background: #C4C4C4;
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbSignUpText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Text*/

position: absolute;
width: 164px;
height: 56px;
left: 112px;
top: 47px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 36px;
line-height: 56px;
display: flex;
align-items: center;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Member Sign Up</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSignUpNameText">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Name: */


position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelEmailText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>243</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Email Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSignUpPasswordText">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>201</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* password text */

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBirthdayText">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*birthday text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Birthday:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSignUpUsernameText">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>163</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Username Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSignUpName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 3 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Password*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSignUpUsername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>158</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 2 */


position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Username*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSignUpPassword">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>197</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 1 */


position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Name*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSignUpEmail">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>236</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 4 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Email*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;s</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="SUSelectDate">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>288</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*boader Date*/
position: absolute;
width: 156px;
height: 37px;
left: 239px;
top: 352px;

background: #EDEDED;
border-radius: 8px; 

/*Text Date*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cBxGender">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>288</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*boader Gender*/
position: absolute;
width: 156px;
height: 37px;
left: 239px;
top: 352px;

background: #EDEDED;
border-radius: 8px; 

/*Text Gender*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenderText">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>290</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Gender Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAdressText">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>340</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Address Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhoneNumberText">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>380</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Address Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSignUpAddress">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>336</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 4 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Email*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRecieveEmailText">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>435</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Address Text*/

position: absolute;
width: 104px;
height: 28px;
left: 65px;
top: 136px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

text-align: center;
letter-spacing: -0.408px;</string>
   </property>
   <property name="text">
    <string>Recieve Emails:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chB_Emails">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>440</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 40px;
height: 40px;
left: 627px;
top: 545px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSignup">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 4 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 10px; 

/*Text Email*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 20px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;
</string>
   </property>
   <property name="text">
    <string>Sign-Up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSignUpPh">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>376</y>
     <width>591</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*Border Sign-up 4 */

position: absolute;
width: 548px;
height: 37px;
left: 202px;
top: 128px;

background: #EDEDED;
border-radius: 8px; 

/*Text Email*/ 

position: absolute;
width: 305px;
height: 31px;
left: 115px;
top: 153px;

font-family: Roboto;
font-style: normal;
font-weight: normal;
font-size: 18px;
line-height: 22px;
/* or 92% */

letter-spacing: -0.408px;

color: #868181;s</string>
   </property>
  </widget>
  <zorder>lbBoarderSignUp</zorder>
  <zorder>lbSignUpText</zorder>
  <zorder>labelSignUpNameText</zorder>
  <zorder>labelEmailText</zorder>
  <zorder>labelSignUpPasswordText</zorder>
  <zorder>labelBirthdayText</zorder>
  <zorder>labelSignUpUsernameText</zorder>
  <zorder>txtSignUpName</zorder>
  <zorder>txtSignUpUsername</zorder>
  <zorder>txtSignUpPassword</zorder>
  <zorder>txtSignUpEmail</zorder>
  <zorder>SUSelectDate</zorder>
  <zorder>cBxGender</zorder>
  <zorder>labelGenderText</zorder>
  <zorder>labelAdressText</zorder>
  <zorder>labelPhoneNumberText</zorder>
  <zorder>labelRecieveEmailText</zorder>
  <zorder>chB_Emails</zorder>
  <zorder>btnSignup</zorder>
  <zorder>txtSignUpAddress</zorder>
  <zorder>txtSignUpPh</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
